src/tables.rs

```diff
@@ -18,12 +18,10 @@ impl Tables {
         }
     }
 
-    pub fn create_row<K: AsRef<str>>(&mut self, table: &str, key: K) -> &mut Row {
+    pub fn create_row(&mut self, table: &str, key: PrimaryKey) -> &mut Row {
         let rows = self.tables.entry(table.to_string()).or_insert(Rows::new());
-        let row = rows
-            .pks
-            .entry(key.as_ref().to_string())
-            .or_insert(Row::new());
+        let key_debug = format!("{:?}", key);
```

Comment:
Should be on the `panic` directly. In the sense, inline this code at line 35 directly.